package/gluon-luci-node-role/i18n/de.po

```diff
@@ -0,0 +1,30 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"PO-Revision-Date: 2015-04-26 17:18+0200\n"
+"Last-Translator: Tobias Hachmer <tobias@hachmer.de>\n"
+"Language-Team: German\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid "Purpose"
+msgstr "Verwendungszweck"
+
+msgid "Node Role"
+msgstr "Knoten Rolle"
```

Comment:
As I suggested to change both "Purpose" and "Node Role" to "Node role", these two would become one. I think we can go with "Verwendungszweck". If we want to keep the translation closer to the English version, this should be "Knotenrolle" or "Knoten-Rolle", not "Knoten Rolle".